packages/wonder-blocks-data/components/data.js

```diff
@@ -106,10 +106,13 @@ export default class Data<TOptions, TData> extends React.Component<
         const {getEntry} = ResponseCache.Default;
 
         const cachedData = getEntry(handler, options);
-        if (!Server.isServerSide() && cachedData == null) {
+        if (
+            !Server.isServerSide() &&
+            (cachedData == null || handler.refreshCache(options, cachedData))
```

Comment:
I was confused by this name - since I assumed that calling it would refresh the cached (and having a side effect inside an `if` is scary), instead it behaves somewhat like `shouldRefreshCache` - having a name like that would make usage like this a bit clearer!